tenets/codelingo/go/global-var/example.go

```diff
@@ -0,0 +1,92 @@
+package code
+
+import (
+	"errors"
+)
+
+const constant = 0
+
+var myVar = 0
+
+var myTestVar = 0
+
+func someCode() bool {
+	yourVar := true
+	return yourVar
+}
+
+var theVar = true
+
+var myVar = "global"
+
+var theVar = 2
+
+func someCode() bool {
+	return true
+}
+
+var myVar = "global"
+
+var myVar1, myVar2 = 1, 2
+
+var _ = 0
+
+// Those are not errors
+var myVar = 1
+
+// Those are fake errors which are currently not detected
+// because they start with 'err' or 'Err' and we don't
+// check if such a variable implements the error interface.
```

Comment:
I don't see anything in the tenet about checking whether the name of the ident starts with `err` or `Err`. Is this comment accurate?

Reply:
It may need to be split into 2 tenets to cover everything.

Reply:
No idea why those test cases were in there. They did come from the external tool though. I'll make sure I'm getting the right test cases.

Reply:
I see. The owner of the original linter must have re-used some test cases from a different linter. But they are actually testing global variables inside the 'error' code.